### ISSUE TITLE
Verify API for mutlifactor authentication

### ISSUE THREAD

**Issue by abdalasif** [TeleSign/php_telesign_enterprise#6]
The verify for multi-factor authentication needs to be added in sdk

**Comment by adohertyTS** [comment]
Hi @abdalasif , I believe this SDK (Telesign PHP Full-service SDK), already has a class you can use to send an SMS OTP for multi-factor authentication. You can find a tutorial explaining how to use it for that here: https://developer.telesign.com/enterprise/docs/sms-verify-api-tutorial-send-one-time-passcode-with-telesign-php-sdk

Does that help you to proceed with your use case?

**Comment by abdalasif** [comment]
@adohertyTS This SDK does not uses the TeleSign new [VERIFY API](https://developer.telesign.com/enterprise/reference/createverificationprocess)

**Comment by adohertyTS** [comment]
@abdalasif That's true, it uses the older SMS Verify API. We do plan to add support for the new Verify API, but we don't have a specific date planned for that work yet.
